client/src/pages/EditProfilePage.css

```diff
@@ -0,0 +1,59 @@
+.title-text{
+    margin: 40px;
+    font-size: 30px;
+}
+
+.icon-text-top{
+    margin-bottom: 10px;
+    font-size: 20px;
+}
+
+.icon-text{
+    margin-top: 50px;
+    margin-bottom: 10px;
+    font-size: 20px;
+}
+
+.float-container {
+    padding: 20px;
+    display: flex;
+}
+
+.float-icon {
+    width: 200px;
+    float: left;
+    padding: 20px;
+    padding-right: 60px;
+    justify-content: "center";
+}
+
+.upload {
+    position: relative;
+    top: 5%;
+    left: 25%;
+}
+
+.submit {
+    position: relative;
+    top: 5%;
+}
+
+.float-div {
+    width: 70%;
+    float: left;
+    padding: 20px;
+}
+
+.avatar {
```

Comment:
let's stay consistent on how we style things. 

In another component (https://github.com/hatchways/team-ninja-turtles/blob/dev/client/src/components/ContestCard.js) for example, we use makeStyles